include/core/utilities.h

```diff
@@ -390,6 +390,14 @@ fill_string_vectors_from_file(
 void
 create_output_folder(const std::string &dirname);
 
+/**
+ * @brief Delete the output folder and its content.
+ *
+ * @param[in] dirname Output directory name.
+ */
+void
+delete_output_folder(const std::string &dirname);
+
```

Comment:
Is this needed?

Reply:
Meaning, do we have an option now to delete the output folder?

Reply:
I use it for my pvtu outputs of the reinitializations steps, in order to conserve only the reinitialization steps of the last time-step (otherwise, if I kept all time-steps, it would demand a folder in folder architecture)
If the previous time-step had more reinitialization steps, the pvtu files would be kept unless they are deleted 

*when I say "time-step", I refer to (global) simulation time-steps not reinitialization steps